packages/workspace/src/schematics/library/library.ts

```diff
@@ -70,10 +70,7 @@ function updateTsConfig(options: NormalizedSchema): Rule {
       }
 
       c.paths[options.importPath] = [
-        maybeJs(
```

Comment:
Did this move somewhere? I think it's still needed right?

Reply:
I was trying to remove the need for scattered `toJS` behaviour having to be sprinkled around the plugin.

I was hoping that changing content in the config files would make this change unnecessary here, but I think since changing it to only newly created files, I need to either operate on modified files (maybe find only lines that changed) and change those to `.js` as well.